demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
I would say that if debugging, you have devtools to highlight things as you need to. I agree with Adam, the demo is for consumers.  Adding that variable is more complexity that I don't think is necessary.